website/docs/r/netapp_volume.html.markdown

```diff
@@ -0,0 +1,129 @@
+  resource_group_name = "${azurerm_resource_group.example.name}"
+  address_space       = ["10.0.0.0/16"]
+}
+
+resource "azurerm_subnet" "example" {
+  name                 = "example-subnet"
+  resource_group_name  = "${azurerm_resource_group.example.name}"
+  virtual_network_name = "${azurerm_virtual_network.example.name}"
+  address_prefix       = "10.0.2.0/24"
+  delegation {
+    name = "testdelegation"
+  
+    service_delegation {
+      name    = "Microsoft.Netapp/volumes"
+      actions = ["Microsoft.Network/networkinterfaces/*", "Microsoft.Network/virtualNetworks/subnets/join/action"]
+    }
+  }
+}
+
+resource "azurerm_netapp_account" "example" {
+  name                = "example-netappaccount"
+  location            = "${azurerm_resource_group.example.location}"
+  resource_group_name = "${azurerm_resource_group.example.name}"
+}
+
+resource "azurerm_netapp_pool" "example" {
+  name                = "example-netapppool"
+  location            = "${azurerm_resource_group.example.location}"
+  resource_group_name = "${azurerm_resource_group.example.name}"
+  account_name        = "${azurerm_netapp_account.example.name}"
+  service_level       = "Premium"
+  size_in_4_tb        = "1"
+}
+
+resource "azurerm_netapp_volume" "example" {
+  name                = "example-netappvolume"
+  location            = "${azurerm_resource_group.example.location}"
+  resource_group_name = "${azurerm_resource_group.example.name}"
+  account_name        = "${azurerm_netapp_account.example.name}"
+  pool_name           = "${azurerm_netapp_pool.example.name}"
+  creation_token      = "my-unique-file-path"
+  service_level       = "Premium"
+  subnet_id           = "${azurerm_subnet.example.id}"
+  usage_threshold     = "100"
+}
+```
+
+## Argument Reference
+
+The following arguments are supported:
+
+* `name` - (Required) The name of the NetApp Volume. Changing this forces a new resource to be created.
+
+* `resource_group_name` - (Required) The name of the resource group where the NetApp Volume should be created. Changing this forces a new resource to be created.
+
+* `location` - (Required) Specifies the supported Azure location where the resource exists. Changing this forces a new resource to be created.
+
+* `account_name` - (Required) The name of the NetApp account in which the NetApp Pool should be created.
+
+* `pool_name` - (Required) The name of the NetApp pool in which the NetApp Volume should be created.
+
+* `creation_token` - (Required) A unique file path for the volume. Used when creating mount targets.
+
+* `service_level` - (Required) The service level of the file system. Valid values include `Premium`, `Standard`, or `Ultra`.
+
+* `subnet_id` - (Required) The Azure Resource URI for a delegated subnet. Must have the delegation Microsoft.NetApp/volumes.
+
+* `usage_threshold` - (Required) Maximum storage quota allowed for a file system in bytes. This is a soft quota used for alerting only.
```

Comment:
Perhaps this field wants to be renamed (since it's being converted to GB's above)?

```suggestion
* `usage_threshold` - (Required) The maximum Storage Quota in Gigabytes allowed for a file system.

-> **NOTE:** This is a soft quota used for alerting only.
```

Reply:
Fixed